book_controller.rb

```diff
@@ -0,0 +1,25 @@
+require 'json'
+require './book'
+
+# bookcontroller
+module BookController
+  def load_books
+    data = []
+    file = './books.json'
+    if File.read(file) != ''
+      JSON.parse(File.read(file)).each do |book|
+        data.push(Book.new(book['Title'], book['Author']))
+      end
+    end
+    data
+  end
```

Comment:
- [ ] please have a look at the below requirement:
- On startup of your app load all data from files. Remember to handle the case when one or more of the files are missing.
when deleting one of the .json files the user is not able to start the application anymore, please make sure your application creates the file in case it doesn't excist.